lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -347,10 +351,10 @@ def inner(self, default_store, module_count, mongo_calls, sql_queries, *args, **
         return inner
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 3, 4, 21),
-        (ModuleStoreEnum.Type.mongo, 20, 4, 21),
-        (ModuleStoreEnum.Type.split, 3, 13, 21),
-        (ModuleStoreEnum.Type.split, 20, 13, 21),
+        (ModuleStoreEnum.Type.mongo, 3, 4, 22),
```

Comment:
In the case of team topics, the additional query is necessary. I wasn't aware of a "no new query" policy. I also don't know if the additional query is actually a performance issue-- I've had the feeling that mongo queries are more expensive than sql queries. Before we try to reduce the existing queries, I would want to know what the performance cost of them is so that we aren't doing premature optimization.